clickhouse/github-repo

```diff
@@ -0,0 +1 @@
+https://github.com/ClickHouse/docker-library
```

Comment:
Mostly an FYI, this will get used for things like links in the "Quick Reference" section to GitHub issues -- is this where you want users to file issues related to the image, or would you rather they go to ClickHouse/ClickHouse?  (do you plan to keep issues enabled on the docker-library repository?)

Reply:
I misinterpret [the readme](https://github.com/docker-library/docs?tab=readme-ov-file#github-repo), it should be the `ClickHouse/ClickHouse` for sure